src/sage/game_theory/normal_form_game.py

```diff
@@ -236,8 +236,7 @@
   `Gambit <http://gambit.sourceforge.net/>`_ [Gambit]_. At present this is
   the only gambit algorithm available in sage but further development will
```

Comment:
Is "available in sage" accurate? Are you just leaving it as is for the follow-up PR?